src/util/primitives.js

```diff
@@ -185,12 +185,26 @@ class Aabb {
         return new Aabb(min, max);
     }
 
+    clone(): Aabb {
+        return new Aabb(vec3.clone(this.min), vec3.clone(this.max));
+    }
```

Comment:
Unused in the PR, should probably be removed?

Reply:
Yes leftover that ended up being unused, I've removed it in latest commits.